README.md

```diff
@@ -420,13 +200,37 @@ The `dosageInstruction` property may contain values with the following propertie
 |timing|optional [Timing](https://hl7.org/fhir/R4B/datatypes.html#timing)|-|When medication should be administered|
 |doseAndRate|list of Element|-|Amount of medication administered|
 |doseAndRate>type|optional [DoseRateType](https://hl7.org/fhir/R4B/codesystem-dose-rate-type.html)|e.g. "calculated", "ordered"||
-|doseAndRate>dose|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|TBD: We should either use "pills" or some weight unit here.|
+|doseAndRate>dose|optional [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|Unit is always "tablet". Value may be 0.5, 1 or 2.|
```

Comment:
changed wording to "Value is usually 0.5, 1 or 2". We do not enforce the limit from a technical side, but Alex said it would usually only be one of those values.

Reply:
Sounds good, that makes sense 👍